### DISCUSSION TITLE
Sound Player Undownloadable

### DISCUSSION THREAD

**Discussion by Niswendel** [jef/streetmerchant#1427] [General]
Hi,
I was reinstalling the software and noticed that with the new website there isn't a download for the sound player. Does anyone know where i can get this.

**Comment by CorgFather** [comment]
What platform are you on? If Windows, [Is this what you mean](https://github.com/jef/streetmerchant/wiki/Installation:-Windows#audio-notifications)?

**Reply by Niswendel** [reply]
i MODT HAVE MISSED IT, THANKS SO MUCH
